desktop/apps/personalize/components/App.js

```diff
@@ -9,13 +9,19 @@ import Budget from '@artsy/reaction-force/dist/Components/Onboarding/Steps/Budge
 
 export class App extends React.Component {
   static propTypes = {
-    currentUser: PropTypes.object
+    currentUser: PropTypes.object,
+    redirectTo: PropTypes.string,
+    forceStep: PropTypes.string
```

Comment:
@kanaabe what's the purpose of these static propTypes again?

Reply:
So our linters are happy! Haha just kidding. It's so we can define types on our props. Similar to TS interface/type definitions but we don't have TS in Force, yet..

Reply:
W have an open PR #2145 once we merge it we’ll be able to use TS, but this is :+1: for now